app/views/shared/_password_policy.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="panel">
+  <div class="password-policy__label">
```

Comment:
No need for a heading style, you can use `.panel__heading`. See [panel style example](https://www.moneyadviceservice.org.uk/en/styleguide/components/common#section-panel).